Tasks/AndroidSigningV2/package.json

```diff
@@ -18,6 +18,6 @@
   "homepage": "https://github.com/microsoft/vsts-tasks#readme",
   "dependencies": {
     "securefiles-common": "file:../../_build/Tasks/Common/securefiles-common-1.0.0.tgz",
-    "vsts-task-lib": "2.1.0"
+    "vsts-task-lib": "^2.4.0"
```

Comment:
We stopped using the compat '^' flag to prevent any accidental behavioral change.  Any reason to put it back in?  Because we're using 'package-lock' now?

Reply:
This is just what `npm install` did for me.  Because this will only let the patch version slide, we shouldn't see any behavior changes as long as the package is versioned correctly.  Are we editing these manually?  That doesn't seem right.

Reply:
Historically, yes.  And, yes, we've been hit (many times) with bugs when this version wasn't pinned.

Reply:
`~` only lets the patch version slide. `^` lets the minor version slide too. When vso-node-api was in preview they made some breaking changes within the minor versions, perhaps because of this misunderstanding of what `^` will match.

Regarding manually, you can pass `--save-exact` to npm. `~` seems reasonable too, to get bug and security fixes.

CC @stephenmichaelf

Reply:
Ok.  I can change it then.  I just thought that this is what we had package-lock.json for.

Reply:
Good point. Ignoring the complications of common it should be easy to test if package-lock prevents inconsistent sliding.

CC @ericsciple